### PR TITLE
Show Tenant fetcher custom metrics on Compass Director Grafana Dashboard

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Show Tenant fetcher custom metrics on Compass Director Grafana Dashboard

**Related issue(s)**
https://github.com/kyma-incubator/compass/issues/1235

### CODE REVIEW

File: resources/monitoring/charts/prometheus-pushgateway/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/prometheus-pushgateway/OWNERS:1]: Please delete this file

File: resources/monitoring/charts/prometheus-pushgateway/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Prometheus Pushgateway
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/prometheus-pushgateway/README.md:1]: We don't put readmes for the subcharts as they can be irrelevant for our use case. Please delete this file.

File: resources/monitoring/charts/prometheus-pushgateway/templates/pdb.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: {{ template "prometheus-pushgateway.fullname" . }}
+  labels:
+{{ template "prometheus-pushgateway.defaultLabels" merge (dict "extraLabels" .Values.podLabels) .  }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "prometheus-pushgateway.name" . }}
+{{ toYaml .Values.podDisruptionBudget | indent 2 }}
+{{- end -}}
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/prometheus-pushgateway/templates/pdb.yaml:13]: new line pls

File: resources/monitoring/charts/prometheus-pushgateway/templates/policy.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Required because Prometheus Operator doesn't have Istio Sidecar
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/prometheus-pushgateway/templates/policy.yaml:1]: Please create a kyma-additions folder and put all the additional files that did not exist on the original chart there.

File: resources/monitoring/requirements.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+    - name: prometheus-pushgateway
+      condition: pushgateway.enabled
+
```
Comment (suleymanakbas91) [inline resources/monitoring/requirements.yaml:4]: Delete unnecessary space and put a new line

File: resources/monitoring/charts/prometheus-pushgateway/.helmignore [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+
+# OWNERS file for Kubernetes
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/prometheus-pushgateway/.helmignore:23]: Delete this when you delete the file below

File: resources/monitoring/values.yaml [hunk 7]
```diff
@@ -1968,3 +1968,6 @@ prometheus:
     ## https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#podmetricsendpoint
     ##
     # podMetricsEndpoints: []
+
+pushgateway:
```
Comment (kfurgol) [inline resources/monitoring/values.yaml:1972]: If the idea suggested above about checking for push gateway in `tenant-fetcher-job` is valid, I believe it should be a global field

**Pull request merged**